addons/hw_drivers/tools/helpers.py

```diff
@@ -0,0 +1,123 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import netifaces
+from pathlib import Path
+import datetime
+from OpenSSL import crypto
+import urllib3
+import json
+import logging
+import subprocess
+
+from odoo import _
+
+_logger = logging.getLogger(__name__)
+
+#----------------------------------------------------------
+# Helper
+#----------------------------------------------------------
+
+def access_point():
+    return get_ip() == '10.11.12.1'
+
+def add_credential(db_uuid, enterprise_code):
+    write_file('odoo-db-uuid.conf', db_uuid)
```

Comment:
don't you need to do your remount brol here?

Reply:
brol ? :)

yes we do in write_file
https://github.com/odoo/odoo/pull/35854/files/d0fdbdc5818d8f8743b11cbcfdb1fefc66c4ccc5#diff-04d7277bee5c8509e43e4f14075426e8R119

Reply:
ok, good for me :)